examples/fabric/v4/cloudRouterConnectivity/MutliCloudConnection/main.tf

```diff
@@ -0,0 +1,218 @@
+provider "equinix" {
+  client_id     = var.equinix_client_id
+  client_secret = var.equinix_client_secret
+}
+
+resource "equinix_fabric_cloud_router" "test"{
+  name = var.fcr_name
+  type = var.fcr_type
+  notifications{
+    type =var.notifications_type
+    emails =var.notifications_emails
+  }
+  order {
+    purchase_order_number = var.purchase_order_number
+  }
+  location {
+    metro_code = var.fcr_location
+  }
+  package {
+    code = var.fcr_package
+  }
+  project {
+    project_id = var.fcr_project
+  }
+ account {
+    account_number = var.fcr_account
+ }
+}
+
+output "fcr_result" {
+  value = equinix_fabric_cloud_router.test.id
+}
+
+data "equinix_fabric_service_profiles" "azure"{
+  filter{
+    property = "/name"
+    operator = "="
+    values   = [var.azure_fabric_sp_name]
+  }
+}
+
+resource  "equinix_fabric_connection" "fcr2azure"{
+  name = var.azure_connection_name
+  type = var.azure_connection_type
+
+  notifications {
+    type   = var.azure_notifications_type
+    emails = var.azure_notifications_emails
+  }
+  bandwidth = var.azure_bandwidth
+  redundancy {
+    priority = var.azure_redundancy
+  }
+  order {
+    purchase_order_number = var.azure_purchase_order_number
+  }
+  a_side {
+    access_point {
+      type = var.azure_aside_ap_type
+      router {
+        uuid = equinix_fabric_cloud_router.test.id
+      }
+    }
+  }
+
+  z_side {
+    access_point {
+      type               = var.azure_zside_ap_type
+      authentication_key = var.azure_zside_ap_authentication_key
+      peering_type       = var.azure_peering_type
+      profile {
+        type = var.azure_zside_ap_profile_type
+        uuid = data.equinix_fabric_service_profiles.azure.id
+      }
+      location {
+        metro_code = var.azure_zside_location
+      }
+    }
+  }
+}
+output "azure_connection_azure" {
+  value = var.azure_connection_name
```

Comment:
Nit: This should be grabbed from the created resource and not just directly from the input variable. It will be the same name but better to have it derived from the correct place.